modules/library/data/src/main/java/org/geotools/data/memory/MemoryDataStore.java

```diff
@@ -26,36 +26,38 @@
 import java.util.Map;
 import java.util.logging.Level;
 
-import org.geotools.data.DataSourceException;
-import org.geotools.data.FeatureReader;
-import org.geotools.data.Query;
+import org.geotools.data.*;
```

Comment:
Looks like your IDE is messing with imports, we try and keep such noise out of commits so we can focus on what is going on.